tests/myst/roles.references.equations.yml

```diff
@@ -0,0 +1,101 @@
+cases:
+  - title: Invalid equation reference
+    mdast:
+      type: root
+      children:
+        - type: paragraph
+          children:
+            - type: text
+              value: 'see '
+            - type: contentReference
+              kind: eq
+              identifier: matrix
+              label: '  matrix  '
+              resolved: false
```

Comment:
@fwkoch I don't think `resolved` is part of the AST? But are part of the transform section?


We can have multiple asts that are parsed, and then brought together.

Reply:
Yeah, I agree with this - will fix it up.